depthai_nodes/ml/parsers/xfeat.py

```diff
@@ -0,0 +1,527 @@
+        Node's input. It is a linking point to which the Neural Network's output is linked. It accepts the output of the Neural Network node.
+    target_input : Node.Input
+        Node's input. It is a linking point to which the Neural Network's output is linked. It accepts the output of the Neural Network node.
+    out : Node.Output
+        Parser sends the processed network results to this output in a form of DepthAI message. It is a linking point from which the processed network results are retrieved.
+    output_layer_feats : str
+        Name of the output layer from which the features are extracted.
+    output_layer_keypoints : str
+        Name of the output layer from which the keypoints are extracted.
+    output_layer_heatmaps : str
+        Name of the output layer from which the heatmaps are extracted.
+    original_size : Tuple[float, float]
+        Original image size.
+    input_size : Tuple[float, float]
+        Input image size.
+    max_keypoints : int
+        Maximum number of keypoints to keep.
+
+    Output Message/s
+    ----------------
+    **Type**: dai.TrackedFeatures
+
+    **Description**: TrackedFeatures message containing matched keypoints with the same ID.
+
+    Error Handling
+    --------------
+    **ValueError**: If the original image size is not specified.
+    **ValueError**: If the input image size is not specified.
+    **ValueError**: If the maximum number of keypoints is not specified.
+    **ValueError**: If the output layer containing features is not specified.
+    **ValueError**: If the output layer containing keypoints is not specified.
+    **ValueError**: If the output layer containing heatmaps is not specified.
+    """
+
+    def __init__(
+        self,
+        output_layer_feats="feats",
+        output_layer_keypoints="keypoints",
+        output_layer_heatmaps="heatmaps",
+        original_size: Tuple[float, float] = None,
+        input_size: Tuple[float, float] = (640, 352),
+        max_keypoints: int = 4096,
+    ):
+        """Initializes the XFeatParser node.
+
+        @param output_layer_feats: Name of the output layer containing features.
+        @type output_layer_feats: str
+        @param output_layer_keypoints: Name of the output layer containing keypoints.
+        @type output_layer_keypoints: str
+        @param output_layer_heatmaps: Name of the output layer containing heatmaps.
+        @type output_layer_heatmaps: str
+        @param original_size: Original image size.
+        @type original_size: Tuple[float, float]
+        @param input_size: Input image size.
+        @type input_size: Tuple[float, float]
+        @param max_keypoints: Maximum number of keypoints to keep.
+        @type max_keypoints: int
+        """
+        super().__init__(
+            output_layer_feats,
+            output_layer_keypoints,
+            output_layer_heatmaps,
+            original_size,
+            input_size,
+            max_keypoints,
+        )
+
+    def build(
```

Comment:
same as for mono, no need

Reply:
Added in [55095b3](https://github.com/luxonis/depthai-nodes/pull/104/commits/55095b38d4b1c80694093331dcb4c303ea22a893).